resources/views/chapters/show.blade.php

```diff
@@ -6,6 +6,11 @@
     option:entity-search:entity-type="chapter"
 @stop
 
+@push('social-meta')
+    <meta property="og:description" content="{{ Str::words($chapter->description, 50, '...') }}">
```

Comment:
Humm, as per this link, the og description can be set to 110 characters: https://stackoverflow.com/a/35817780/903324

Reply:
Changed to utilize `Str::length` helper with length set to 100.